### ISSUE TITLE
no way to add a theme without blowing away languages?

### ISSUE THREAD

**Issue by kaolin** [sharkdp/bat#172]
It would be really nice to be able to either add a theme to the existing themes, or at least be able to create your own set of themes without losing all of the languages/syntaxes....

I'll admit I may be doing something wrong.

Installed bat via homebrew. Followed instructions to create appropriate theme directory and added a theme. Couldn't reinit cache without creating a syntax directory. Reiniting cache then made it so I had just the two themes I had added, and no languages....

**Comment by kaolin** [comment]
(I did realize that all the source syntaxes/languages were git submodules, so `git submodule init`, `git submodule update` will at least pull in the information to copy over... if you've `git clone`d the repo, versus using the tarball....)

**Comment by sharkdp** [comment]
Unfortunately, there is currently no way to do this (I would suggest to follow what you have done in your second comment).

I think it is a good idea to support this in general. I'll leave this ticket open as a feature request.

**Comment by Calinou** [comment]
This could be tangentially related to this issue, but I am unable to install a syntax file (https://github.com/beefsack/GDScript-sublime) if I don't install any themes at the same time. If I create an empty `themes` directory, then no themes are available at all.

```
❯ bat cache --init
[bat error]: Could not load themes from '/home/hugo/.config/bat/themes'

❯ tree -L 2 $(bat cache --config-dir)
/home/hugo/.config/bat
└── syntaxes
    └── GDScript-sublime
```

**Comment by sharkdp** [comment]
This has been fixed in [bat 0.5](https://github.com/sharkdp/bat/releases/tag/v0.5.0). Please see the release notes and README for details.
